src/onelogin/saml2/settings.py

```diff
@@ -328,6 +347,11 @@ def _add_default_values(self):
         self._sp.setdefault('x509cert', '')
         self._sp.setdefault('privateKey', '')
 
+        # Set the default filenames for the certificates and keys
+        self._idp.setdefault('cert_filename', 'idp.crt')
```

Comment:
Better inject them in the _add_default_values method

Reply:
But it is already part of the `_add_default_values` method